pandas/io/pytables.py

```diff
@@ -929,28 +935,30 @@ def func(_start, _stop, _where):
 
     def put(self, key, value, format=None, append=False, **kwargs):
         """
-        Store object in HDFStore
+        Store object in HDFStore.
 
         Parameters
         ----------
         key      : object
         value    : {Series, DataFrame}
-        format   : 'fixed(f)|table(t)', default is 'fixed'
-            fixed(f) : Fixed format
-                       Fast writing/reading. Not-appendable, nor searchable
-            table(t) : Table format
+        format   : 'Fixed(f)|Table(t)', default is 'fixed'
```

Comment:
If you don't mind, can you leave just one space in each side of the colon? I know it's from the original code, not your changed, but would be good to have this fixed.